app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -20,11 +20,17 @@ module AuthnJwt
     EXP_CLAIM_NAME = "exp"
     NBF_CLAIM_NAME = "nbf"
     IAT_CLAIM_NAME = "iat"
-    SUPPORTED_ALGORITHMS = %w[RS256 RS384 RS512].freeze
+    JTI_CLAIM_NAME = "jti"
+    RSA_ALGORITHMS = %w[RS256 RS384 RS512].freeze
+    ECDSA_ALGORITHMS = %w[ES256 ES384 ES512].freeze
```

Comment:
@nessiLahav  did you rebase? this is a change that Sasha did on purpose..

Reply:
Yes i fixed it thanks